index.html

```diff
@@ -296,9 +301,35 @@ <h3>Skills</h3>
             </div>
             </div>
             </section>
-        
-    
+            <section id="contact-form">
```

Comment:
- [ ] As the previous reviewer mentioned, kindly fix the ```about``` and ```contact``` sections by using the right margin and padding as in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
- [ ] [OPTIONAL] The About lists (Language, Frameworks, Skills) should be centered, kindly implement this
- [ ] [OPTIONAL] You can consider using the same padding/margin for your design, so it can be as close as possible to the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220604_101604](https://user-images.githubusercontent.com/25227022/171993026-35ce983c-1492-45dd-a6bc-028ca31184f3.png)
![Screenshot_20220604_101618](https://user-images.githubusercontent.com/25227022/171993027-89fe5473-8e89-48fe-ab7f-f43b0e31fcd6.png)|![171872898-9eba2174-3aab-4098-bcd9-3a24d1d479fc](https://user-images.githubusercontent.com/25227022/171993032-e7f881ff-6a07-4c9a-a629-4f4dcb36ac68.png)|

Reply:
the one shown your review is the old one, it's already implemented,, kindly review the latest one

Reply:
Hi @r-ahmed2022, I see that you added the changes at the same time when I was finishing the review, kindly don't make changes after you submit the project for review.
You can send the project for review in the dashboard.

Reply:
Ok, thank you. Done already